README.md

```diff
@@ -1,22 +1,18 @@
 # Xapian documentation sprint
 
-We're writing new introductory documentation for Xapian.  See the formatted
-output at https://getting-started-with-xapian.readthedocs.org/
+This is the source for [Xapian's user
+guide](https://getting-started-with-xapian.readthedocs.org/).
+Eventually this repository will be merged into the main Xapian tree.
 
 Join in on channel #xapian on irc.freenode.net (webchat link:
-https://webchat.freenode.net/?channels=%23xapian).
+https://webchat.freenode.net/?channels=%23xapian) or via [our
+mailing lists](https://xapian.org/lists).
 
-This repository will be merged into the main Xapian tree and disappear
-once ready.
-
-You will need the Sphinx documentation tool installed to process the
-documentation.  Confusingly there are several open source projects called
-"Sphinx" - see http://sphinx-doc.org/install.html for tips on installing
-the one you want (installing the `python-sphinx` package should work on
-at least Debian, Fedora and Ubuntu).  We're unsure of the exact minimum version
-requirement, but it's at least Sphinx 1.0.0 (the oldest we've tested with at
-all recently was Sphinx 1.1.3).  Either the Python 2 or Python 3 version should
-work.
+You will need the [Sphinx documentation tool](http://sphinx-doc.org/)
+installed to process the documentation. You can install the `python-sphinx`
+package on Debian, Fedora and Ubuntu, or `pip install -r requirements.txt`
+to install the python package directly. We currently support up to Sphinx
+1.8.5, because we have yet to migrate our custom domain to python3.
```

Comment:
I'm not sure what "custom domain" means here...

Reply:
Ah, http://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html presumably.